tests/urlpatterns_reverse/tests.py

```diff
@@ -200,17 +200,21 @@ def test_prefix_braces(self):
                reverse('non_path_include', prefix='/{{invalid}}/'))
 
     def test_prefix_parenthesis(self):
-        self.assertEqual('/bogus%29/includes/non_path_include/',
+        # Parentheses are allowed and should not cause errors or be escaped
+        self.assertEqual('/bogus)/includes/non_path_include/',
```

Comment:
care to add a separate commit which normalizes these assertions to use the hanging style like you've done in the new test?